bembelimen/joomla-cms
# Fixed error messages

Pull Request for Issue # .

### Summary of Changes
1. When trying to delete the default state an error message was thrown, displaying that "item must be published first". This is clearly the wrong message. Replaced by "you are trying to delete default item".
2. After failed delete a confirmation message was displayed "0 items deleted". Fixed code so now only if the number of items deleted (moved to trash) is greater than 0 a message will be displayed.

### Testing Instructions
* Create a state, set it to "published" and set it default
* Try to delete the default state, take a look at the error message you receive (must be published first)
* Notice the green confirmation message saying that "0 items were deleted" (unnecessary)


### Expected result
User needs to be informed that he / she is trying to delete the default item. Message that "item must be published first" is useless.
Furthermore if no items were deleted a confirmation message about that process is confusing. A message should only be displayed if at least 1 item was moved to trash.

### Actual result
False error & confirmation message.


### Documentation Changes Required
None.
